controller.html

```diff
@@ -0,0 +1,248 @@
+        navigator.mozVibrate ||
+        navigator.msVibrate;
+
+      var transitions = {
+        LOBBY: {
+          entryAction: () => {
+            if(screen === "LOBBY") return;
+            document.getElementById("lobby").style.display = "block";
+            document.getElementById("game_mode").style.display = "none";
+            document.getElementById("device-silenced").style.display = "none";
+            this.screen = "LOBBY";
+          },
+          exitAction: () => {},
+        },
+        GAME: {
+          entryAction: () => {
+            if(screen === "GAME") return;
+            document.getElementById("lobby").style.display = "none";
+            document.getElementById("game_mode").style.display = "block";
+            document.getElementById("device-silenced").style.display = "none";
+            this.screen = "GAME";
+          },
+          exitAction: () => {},
+        },
+        SILENCED: {
+          entryAction: () => {
+            if (screen === "SILENCED") return;
+            document.getElementById("lobby").style.display = "none";
+            document.getElementById("game_mode").style.display = "none";
+            document.getElementById("device-silenced").style.display = "block";
+            this.screen = "SILENCED";
+          },
+          exitAction: () => {
+          },
+        }
+      };
+
+      var airconsole;
+      var screen = "SILENCED";
+      /**
+       * Sets up the communication to the screen.
+       */
+      function init() {
+        airconsole = new AirConsole({ orientation: "portrait", silence_players: true });
+
+        /*
+         * Checks if this device is part of the active game.
+         */
+        airconsole.onActivePlayersChange = function (player) {
+          console.info(`ActivePlayersChange [${ airconsole.device_id }]: player number`, player);
+          var div = document.getElementById("game_output");
+          if (player !== undefined) {
+            div.innerHTML = ["Left Player", "Right Player"][player];
+          } else {
+            // TODO(marc): Remove this block when the controller handles silencing itself
+            // We are not in the lobby but were excluded in which case the controller will show an overlay to inform the user - for now we need to 'fake this' here.
+            if (airconsole.getActivePlayerDeviceIds() > 0) {
+              transitions["SILENCED"].entryAction();
+            } else {
+              // We are in the lobby, in this case there are no active players
+              transitions["LOBBY"].entryAction();
+            }
+          }
+        };
+
+        // We lose master controller status if the current master is not hero but a hero joins the session.
+        airconsole.onConnect = function(device_id) {
+          // TODO(marc): Remove this block when the controller handles silencing itself
+          if(this.receiverIsSilenced_(this.getDeviceId())) {transitions["SILENCED"].entryAction();  return; }
```

Comment:
2 things: you have a todo and the formatting is inline

Reply:
The ToDo in the controller is intended.
When the next task in this chain is done, the intend is to remove this todo and see if it just works as it should.

Before 1.9.0 leaves alpha, the screen, controller and ttf would (at least thats my plan) be removed again and the example updated but for development these should be in here to reflect the 'current state' to test and validate.